fastpath/docker-compose.yml

```diff
@@ -0,0 +1,42 @@
+services:
+  fastpath:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    container_name: ooni-fastpath 
+    ports:
+      - "5000:5000"
```

Comment:
What is listening on port 5000? I tried starting the docker container, but I get connection reset to port 5000, suggesting that perhaps there is no service listening there?